packages/frontend/src/utils/project/contracts-and-permissions/toVerificationStatus.ts

```diff
@@ -1,8 +1,16 @@
-export type VerificationStatus = 'verified' | 'unverified' | 'not-verifiable'
+export type VerificationStatus =
+  | 'verified'
+  | 'unverified'
+  | 'not-verifiable'
+  | 'verification-change'
```

Comment:
This should be named better. Maybe `became-verified`? Same with `projectChangeReport`